docs/kind-ipv6-dualstack.md

```diff
@@ -0,0 +1,280 @@
+# Running OVN-Kubernetes with IPv6 or Dual-stack In KIND
+
+The [KIND](https://github.com/kubernetes-sigs/kind) deployment is used for
+reproducing an OpenShift networking environment with upstream K8S. The value
+proposition is really for developers who want to reproduce an issue or test a
+fix in an environment that can be brought up locally and within a few minutes.
+
+## KIND with IPv6
+
+### Docker Changes For IPv6
+
+To use IPv6, IPv6 needs to be enable in Docker on the host:
```

Comment:
We removed this limitation in the last kind version 0.8.1, meanwhile, the host has IPv6 enabled it automatically creates a bridge with IP6 enabled. There is no need to configure the docker daemon anymore

Reply:
OVN-Kubernetes CI is currently using Kubernetes v1.17.2 and KIND v0.7.0. So I added a comment to indicate this step is only needed for KIND v0.7.0 and older version.